app/controllers/submissions_controller.rb

```diff
@@ -158,10 +158,20 @@ def first_step?
     step == steps.first
   end
 
-  helper_method :material_submission, :last_step?, :first_step?, :material_schema, :labware_at_index
+  helper_method :material_submission, :last_step?, :first_step?, :material_schema, :labware_at_index, :hmdmc_info
 
 private
 
+  def hmdmc_info
+    "<p>Commercially available human cell lines are defined as "\
+    "human cells that have been cultured and have divided "\
+    "outside the body, and have been purchased from a "\
+    "recognised commercial supplier.</p>\n"\
+    "<p>This definition does not include, for example, "\
+    "cell lines obtained from a collaborating academic "\
+    "institution.</p>".html_safe
+  end
+
```

Comment:
As in this case this method `hmdmc_info` is not needed by the controller, just by the view, I'd move it to helpers/submission_helper.rb.